src/System.Device.Gpio/System/Device/Spi/SpiConnectionSettings.cs

```diff
@@ -11,10 +11,6 @@ namespace System.Device.Spi
     /// </summary>
     public sealed class SpiConnectionSettings
```

Comment:
I wonder if we should rename this to `HardwareSpiSettings`, some of these options don't apply for SoftwareSpi

Reply:
They are kind of a mixture.  Better yet... how about just SpiSettings (also I2cSettings)?

Reply:
I don't think we should rename this. At least not to HardwareSpiSettings, SpiSettings might be an option, but honestly I don't think that the current names are wrong, plus a good thing is that they are the same in WinRT, so having parity may be better.